packages/amplify-provider-awscloudformation/resources/overrides-resource/package.json

```diff
@@ -9,7 +9,8 @@
     },
     "dependencies": {
       "@types/fs-extra": "^9.0.11",
-      "fs-extra": "^9.1.0"
+      "fs-extra": "^9.1.0",
+      "@aws-amplify/cli-overrides-helper": "ext8"
```

Comment:
shouldn't this be `"@aws-amplify/cli-overrides-helper": "1.0.0"`?

Reply:
Can the `@types/fs-extra` package can be included in devDependencies instead of dependencies?